### ISSUE TITLE
backup pool_list

### ISSUE THREAD

**Issue by edict3d** [fireice-uk/xmr-stak-cpu#497]
etn-us-west1.nanopool.org:13333 crashed a few times so I tried to add backups.  I thought it was supposed to look like this:

"pool_list" :
[
	{"pool_address" : "etn-us-west1.nanopool.org:13333", "wallet_address" : "etnkQA5h7TcAFoi1HbvQZLMLC329oi9e5Va9hdoDjFuDSH4XrdKVskj2grPmxjBZEE85wUnE9ZSrgWYMSai4RFGT5Trtz8NsJV.edicted/nateboyd@hotmail.com", "rig_id" : "", "pool_password" : "", "use_nicehash" : false, "use_tls" : false, "tls_fingerprint" : "", "pool_weight" : 1 },

],
[
	{"pool_address" : "etn-us-west1.nanopool.org:13333", "wallet_address" : "etnkQA5h7TcAFoi1HbvQZLMLC329oi9e5Va9hdoDjFuDSH4XrdKVskj2grPmxjBZEE85wUnE9ZSrgWYMSai4RFGT5Trtz8NsJV.edicted/nateboyd@hotmail.com", "rig_id" : "", "pool_password" : "", "use_nicehash" : false, "use_tls" : false, "tls_fingerprint" : "", "pool_weight" : 1 },

],

I finally figured out that it looks like this:

"pool_list" :
[
	{"pool_address" : "etn-us-west1.nanopool.org:13333", "wallet_address" : "etnkQA5h7TcAFoi1HbvQZLMLC329oi9e5Va9hdoDjFuDSH4XrdKVskj2grPmxjBZEE85wUnE9ZSrgWYMSai4RFGT5Trtz8NsJV.edicted/nateboyd@hotmail.com", "rig_id" : "", "pool_password" : "", "use_nicehash" : false, "use_tls" : false, "tls_fingerprint" : "", "pool_weight" : 1 },
	{"pool_address" : "etn-us-east1.nanopool.org:13333", "wallet_address" : "etnkQA5h7TcAFoi1HbvQZLMLC329oi9e5Va9hdoDjFuDSH4XrdKVskj2grPmxjBZEE85wUnE9ZSrgWYMSai4RFGT5Trtz8NsJV.edicted/nateboyd@hotmail.com", "rig_id" : "", "pool_password" : "", "use_nicehash" : false, "use_tls" : false, "tls_fingerprint" : "", "pool_weight" : 1 },
	{"pool_address" : "etn-eu1.nanopool.org:13333", "wallet_address" : "etnkQA5h7TcAFoi1HbvQZLMLC329oi9e5Va9hdoDjFuDSH4XrdKVskj2grPmxjBZEE85wUnE9ZSrgWYMSai4RFGT5Trtz8NsJV.edicted/nateboyd@hotmail.com", "rig_id" : "", "pool_password" : "", "use_nicehash" : false, "use_tls" : false, "tls_fingerprint" : "", "pool_weight" : 1 },
	{"pool_address" : "etn-eu2.nanopool.org:13333", "wallet_address" : "etnkQA5h7TcAFoi1HbvQZLMLC329oi9e5Va9hdoDjFuDSH4XrdKVskj2grPmxjBZEE85wUnE9ZSrgWYMSai4RFGT5Trtz8NsJV.edicted/nateboyd@hotmail.com", "rig_id" : "", "pool_password" : "", "use_nicehash" : false, "use_tls" : false, "tls_fingerprint" : "", "pool_weight" : 1 },
],

So basically the regular bracket [], gets ignored and you add to the curly brackets {}, 
I hope this helps someone.

**Comment by psychocrypt** [comment]
this is the old not maintained cpu miner you should open an issue in https://github.com/fireice-uk/xmr-stak/
